src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -1040,3 +1040,7 @@ def set_cookies(url_prefix, token):
     cherrypy.response.cookie['token']['HttpOnly'] = True
     cherrypy.response.cookie['token']['path'] = '/'
     cherrypy.response.cookie['token']['SameSite'] = 'Strict'
+
+
+def delete_token_cookie():
```

Comment:
What do you think?
```suggestion
def delete_cookie_token():

Reply:
Changed it. Thanks @nizamial09

Reply:
but I still see the name as delete_token_cookie()

Reply:
I changed it again as to keep it similar to set_token_cookie and also @alfonsomthd favored  `delete_token_cookie` too. Is that fine with you?

Reply:
Ofcourse it is.